test/asammdf/gui/widgets/test_PlotWidget_Shortcuts.py

```diff
@@ -231,3 +236,264 @@ def test_Plot_Plot_Shortcut_Key_R(self):
+        elif plot.plot.x_axis.grid and  not plot.plot.y_axis.grid:
+            # press key "G"
+            QtTest.QTest.keyClick(plot.plot.viewport(), QtCore.Qt.Key_G)
+            self.processEvents()
+            self.assertTrue(plot.plot.x_axis.grid)
+            self.assertTrue(plot.plot.y_axis.grid)
+            # press key "G"
+            QtTest.QTest.keyClick(plot.plot.viewport(), QtCore.Qt.Key_G)
+            self.processEvents()
+            self.assertFalse(plot.plot.x_axis.grid)
+            self.assertFalse(plot.plot.y_axis.grid)
+            # press key "G"
+            QtTest.QTest.keyClick(plot.plot.viewport(), QtCore.Qt.Key_G)
+            self.processEvents()
+            self.assertTrue(plot.plot.x_axis.grid)
+            self.assertFalse(plot.plot.y_axis.grid)
+        else:
+            # press key "G"
+            QtTest.QTest.keyClick(plot.plot.viewport(), QtCore.Qt.Key_G)
+            self.processEvents()
+            self.assertTrue(plot.plot.x_axis.grid)
+            self.assertFalse(plot.plot.y_axis.grid)
+            # press key "G"
+            QtTest.QTest.keyClick(plot.plot.viewport(), QtCore.Qt.Key_G)
+            self.processEvents()
+            self.assertTrue(plot.plot.x_axis.grid)
+            self.assertTrue(plot.plot.y_axis.grid)
+            # press key "G"
+            QtTest.QTest.keyClick(plot.plot.viewport(), QtCore.Qt.Key_G)
+            self.processEvents()
+            self.assertFalse(plot.plot.x_axis.grid)
+            self.assertFalse(plot.plot.y_axis.grid)
+
+    def test_Plot_Plot_Shortcut_Key_W(self):
+        """
+        Check if signal is fitted properly after pressing key "W".
+        Events:
+            - Open 'FileWidget' with valid measurement.
+            - Create a plot window and load 2 signals
+            - Press key "S" and "I"
+            - Press key "W"
+        Evaluate:
+            - Evaluate that window is created
+            - Evaluate that there are at least one column with first signal color
+            - Evaluate first and last columns where is first signal:
+                * first column after pressing "S" and "I" is full black => signal colors are not there
+                * signal is zoomed => is extended to left side => in last column is both signals colors
+            - Evaluate that after pressing key "W" in first and last column is displayed both signals
+        """
+        # Event
+        self.setUpFileWidget(measurement_file=self.measurement_file, default=True)
+        # Press PushButton "Create Window"
+        self.create_window(window_type="Plot")
+        self.assertEqual(len(self.widget.mdi_area.subWindowList()), 1)
+        plot = self.widget.mdi_area.subWindowList()[0].widget()
+        channel_0 = self.add_channel_to_plot(plot=plot, channel_index=7)
+        channel_1 = self.add_channel_to_plot(plot=plot, channel_name="ASAM.M.SCALAR.UBYTE.HYPERBOLIC")
+        self.assertEqual(2, plot.channel_selection.topLevelItemCount())
```

Comment:
This block seems to be reused often. Maybe you can include it in 'setUp' if it's common for all test cases.